_gsocproposals/2019/proposal_Phoenix.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Development of an experiment independent javascript event display framework and data format
+layout: gsoc_proposal
+project: HSF
+year: 2019
+organization: HSF
```

Comment:
Can you add as orgs:
- University of Massachusetts
- University of Pittsburg
See the layout in any of the _gsocorgs/2019
Then reference them here rather than HSF:
```
organisation:
  - UM
  - UP
```

Reply:
The reason I did this was when I checked the GSoC rules, there were requirements on organisations which I (and probably Ric) can't legally fulfil. I discussed with @graeme-a-stewart that using HSF would be appropriate (phoenix is under the HSF github).

Reply:
Hi @EdwardMoyse ,

That's fine, it is up to the mentors to decide about organizations and affiliations related to the projects and proposals they submit. 

However, in this particular case I think it would be better to define `Phoenix` as the project and keep `HSF` as the organization, since this proposal will contribute to develop Phoenix while HSF is not a *product* itself.

What do you think?

Reply:
Hi @JavierCVilla  - I think this makes sense (though BTW https://github.com/HSF/hsf.github.io/blob/master/_gsocproposals/2019/proposal_PODIOHDF5.md has HSF as the project!)

But I'm adding a new project now, and will push the new version ASAP.

Reply:
@EdwardMoyse You are completely right, IMHO I think Phoenix fits better as a project than HSF, but given the precedent on previous years, please feel free to choose your preferred option.